### ISSUE TITLE
The cellcoal version used in this study doesn't seem to be available (simulated bulk data request)

### ISSUE THREAD

**Issue by tomouellette** [cbg-ethz/scSomMerClock#1]
Hi there,

I was wondering if an updated version (1.1.2) of [cellcoal](https://github.com/dapogon/cellcoal/releases) is going to be posted with this paper. It seems like the current version (1.1.1) that is available for download is not compatible with this pipeline.

I believe the missing argument is the tree rate switches argument:

```bash
[GENEALOGY MODIFIERS]
    [root branch length ratio] k0.5
    [outgroup branch length ratio] q1
    [rate variation among branches] 
    **[tree rate switches] N1 2.0**
```

The reason I ask about this is that I noticed that you ran both the `neutralitytestr` and `mobster` as a benchmark against bulk, but I'd be interested in running some of my recent approaches ([Ouellette & Awadalla, PLoS Comp. Bio., 2022](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010007)) and some of my new, ongoing and not fully released bulk-level methods ([Ouellette & Awadalla, ICML AI 4 Science, 2022](https://openreview.net/pdf?id=bQ4cfdd0KS)) on your bulk simulated and, possibly, real data as well. The reference would be appreciated as well since we showed that we are very competitive if not better than single statistics and/or mixture models for detecting selection in bulk data (show other things to like binomial mixtures overestimate and others underestimate, etc; making that more evident in follow up work).

Thanks,
Tom O.

(P.S. I'd like to also throw out there that I really like the cellcoal framework -- it's fast and seems to capture a lot of the underlying variation in single-cell data, I'd be glad to help out with making the code base a bit more user-friendly if that's of any interest e.g. wrapping some python around it/redesigning the CLI interface, etc.)

**Comment by NBMueller** [comment]
Dear Tom,

David Posada (Github: dapogon), not me, implemented CellCoal and added the rate switch for this project; it should be available in the latest CellCoal version (v1.1.1).
I think he would appreciate a hand with a more user-friendly code-base, why don't you post that offer in the CellCoal Github directly?

Cheers and best,
Nico

**Comment by tomouellette** [comment]
Thanks Nico, this was helpful!

From looking back at the `cellcoal` documentation, it looks like `tree rate switches` wasn't implemented until v1.3.0 (see here: https://github.com/dapogon/cellcoal/blob/4d3f513a4fd409f9fee43ce811f74f39cce0717a/src/cellcoal.c#L120). 

Interestingly, in the simulations config within `scSomMerClock`, the specified `cellcoal` version is actually 1.2.0 (see here: https://github.com/cbg-ethz/scSomMerClock/blob/main/configs/simulations/config.simulations_noClock_bulk.yaml#L5)

I guess the `cellcoal` releases aren't up to date with the current development version but this issue is resolved if the cellcoal repository is cloned (instead of downloading the latest release).
